flutter/lib/src/view_hierarchy/view_hierarchy_event_processor.dart

```diff
@@ -1,16 +1,26 @@
+import 'dart:async';
+
 import '../../sentry_flutter.dart';
+import '../utils/debouncer.dart';
 import 'sentry_tree_walker.dart';
 
 /// A [EventProcessor] that renders an ASCII representation of the entire view
 /// hierarchy of the application when an error happens and includes it as an
 /// attachment to the [Hint].
 class SentryViewHierarchyEventProcessor implements EventProcessor {
-  SentryViewHierarchyEventProcessor(this._options);
-
   final SentryFlutterOptions _options;
+  late final Debouncer _debouncer;
+
+  SentryViewHierarchyEventProcessor(this._options) {
+    _debouncer = Debouncer(
+      // ignore: invalid_use_of_internal_member
+      _options.clock,
+      waitTimeMs: 2000,
```

Comment:
been only seeing this now but I think it's nicer and more 'dart-y' if we used a Duration instead of an int to represent the time.

so we can do `waitTime: Duration(seconds: 2)`